digitalpublicgoods/fedora-linux.json

```diff
@@ -0,0 +1,110 @@
+        "first and last name",
+        "title and your company's name",
+        "home, billing, or other physical address (including street name, name of a city or town, state/province)",
+        "country code",
+        "e-mail address",
+        "telephone number",
+        "any other identifier that permits Fedora to make physical or online contact with you"
+      ],
+      "thirdPartyDataSharing": "Yes",
+      "dataSharingCircumstances": [
+        "Unless you consent, Fedora will never process or share the personal data you provide to us except as described below. Fedora may share your personal data with third parties under any of the following circumstances:",
+        "Your publicly available personal data in the Fedora account system, as described above, is accessible by anyone unless you, as the account holder, opt out as already described in this Privacy Statement.",
+        "As required to provide service, and for e-mail housing (as a consequence of uses already described in this Privacy Statement). It is in Fedora’s legitimate business interest to provide all users an accurate record of data and content provided by Fedora’s services, and to maintain the integrity of that data and content for historical, scientific, and research purposes. This data and content may include but is not limited to email, code changes, comments, and artifacts.",
+        "As required by law (such as responding to a valid subpoena, warrant, audit, or agency action, or to prevent fraud).",
+        "For research activities, including the production of statistical reports (such aggregated information is used to describe our services and is not used to contact the subjects of the report)."
+      ],
+      "ensurePrivacySecurity": "Yes",
+      "privacySecurityDescription": "Fedora trains its administrators on our privacy policy guidelines and makes our privacy policy available to our partners. Our website uses Secure Socket Layer (SSL) technology, which encrypts your personal data when you send your personal information on our website. In addition, Fedora and its partners enter into confidentiality agreements which require that care and precautions be taken to prevent loss, misuse, or disclosure of your personal data."
+    },
+    "inappropriateIllegalContent": {
+      "collectStoreDistribute": "No",
+      "type": "",
+      "contentFilter": "",
+      "policyGuidelinesDocumentationLink": "",
+      "illegalContentDetection": "",
+      "illegalContentDetectionMechanism": ""
+    },
+    "protectionFromHarassment": {
+      "userInteraction": "No",
+      "addressSafetySecurityUnderageUsers": "",
+      "stepsAddressRiskPreventSafetyUnderageUsers": [
+        ""
+      ],
+      "griefAbuseHarassmentProtection": "Yes",
+      "harassmentProtectionSteps": [
+        ""
+      ]
+    }
+  },
+  "locations": {
+    "developmentCountries": [
+      "United States of America"
+    ],
+    "deploymentCountries": [
```

Comment:
This list looks too short for listing countries where Fedora is in use. Curious to hear the rationale for listing these countries specifically.